lightning/mydump/loader.go

```diff
@@ -40,11 +38,23 @@ type MDTableMeta struct {
 	DB         string
 	Name       string
 	SchemaFile string
-	DataFiles  []string
+	DataFiles  []*SourceFileMeta
 	charSet    string
 	TotalSize  int64
 }
 
+type SourceMeta struct {
+	Path        string
+	Type        SourceType
+	Compression Compression
+	SortKey     string
+}
+
+type SourceFileMeta struct {
+	SourceMeta
+	Size int64
+}
```

Comment:
i don't think you need this type...

Reply:
Yes, I update this by using mydump.fileInfo. But this end up with exposing fileInfo to FileInfo because of using in restore